services/storage/tests/unit/test__legacy_storage_sdk_compatibility.py

```diff
@@ -56,10 +55,12 @@ def location_name() -> str:
     return SimcoreS3DataManager.get_location_name()
 
 
+@pytest.mark.skip(reason="legacy service")
```

Comment:
Q: could you ellaborate more why this is disabled, I did not quite get the reason. Maybe it would be helpful to have a note here for the future.
I find these short reasons not that suseful in skipped tests

Reply:
actually I will re-enable it. I need to find a way to do this as this uses the aiohttp client.
I will probably have to start the real storage in order to run that

Reply:
fixed